<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherWindow</class>
 <widget class="QMainWindow" name="LauncherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Launcher</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="Name_server">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>300</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 18pt &quot;Bahnschrift Light Condensed&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;Linage 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1277</width>
      <height>728</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame
{background-image: url(:/images/background.png);
background-repeat: no-repeat;
                background-position: center;
                border: none;  
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="Telegram">
     <property name="geometry">
      <rect>
       <x>1232</x>
       <y>110</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#Telegram {
  width: 32px;
  height: 32px;
  border-radius: 16px; /* Половина высоты кнопки для создания круглой формы */
  background-color: black;
  border: 2px solid #ffffff;
  padding: 0;
 
	background-image: url(:/images/telegram.png);
  background-position: center;
 
}


#Telegram:hover:pressed {
    background-color: rgba(92, 184, 92, 80%); /* Прозрачный фон кнопки при нажатии */
}

#Telegram:hover:!pressed {
 background-color: black;
	 border: 2px solid #ffca00;
  padding: 0;
	
	background-image: url(:/images/telegram_hover.png);
}


</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="Facebook">
     <property name="geometry">
      <rect>
       <x>1232</x>
       <y>160</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#Facebook{
  width: 32px;
  height: 32px;
  border-radius: 16px; /* Половина высоты кнопки для создания круглой формы */
  background-color: black;
  border: 2px solid #ffffff;
  padding: 0;
	background-image: url(:/images/facebook.png);
  background-position: center;
 
}


#Facebook:hover:pressed {
    background-color: rgba(92, 184, 92, 80%); /* Прозрачный фон кнопки при нажатии */
}

#Facebook:hover:!pressed {
 background-color: black;
	 border: 2px solid #ffca00;
  padding: 0;
	
	background-image: url(:/images/facebook_hover.png);
}


</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="VK">
     <property name="geometry">
      <rect>
       <x>1232</x>
       <y>210</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#VK{
  width: 32px;
  height: 32px;
  border-radius: 16px; /* Половина высоты кнопки для создания круглой формы */
  background-color: black;
  border: 2px solid #ffffff;
  padding: 0;
	background-image: url(:/images/vk.png);
  background-position: center;
 
}


#VK:hover:pressed {
    background-color: rgba(92, 184, 92, 80%); /* Прозрачный фон кнопки при нажатии */
}

#VK:hover:!pressed {
 background-color: black;
 border: 2px solid #ffca00;
  padding: 0;	
	
	background-image: url(:/images/vk_hover.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="Instagram">
     <property name="geometry">
      <rect>
       <x>1232</x>
       <y>260</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#Instagram{
  width: 32px;
  height: 32px;
  border-radius: 16px; /* Половина высоты кнопки для создания круглой формы */
  background-color: black;
  border: 2px solid #ffffff;
  padding: 0;
	background-image: url(:/images/instagram.png);
  background-position: center;
 
}


#Instagram:hover:pressed {
    background-color: rgba(92, 184, 92, 80%); /* Прозрачный фон кнопки при нажатии */
}

#Instagram:hover:!pressed {
 background-color: black;
 border: 2px solid #ffca00;
  padding: 0;	
	
	background-image: url(:/images/instagram_hover.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="Discord">
     <property name="geometry">
      <rect>
       <x>1232</x>
       <y>310</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#Discord{
  width: 32px;
  height: 32px;
  border-radius: 16px; /* Половина высоты кнопки для создания круглой формы */
  background-color: black;
  border: 2px solid #ffffff;
  padding: 0;
	background-image: url(:/images/discord.png);
  background-position: center;
 
}


#Discord:hover:pressed {
    background-color: rgba(92, 184, 92, 80%); /* Прозрачный фон кнопки при нажатии */
}

#Discord:hover:!pressed {
 background-color: black;
 border: 2px solid #ffca00;
  padding: 0;	
	
	background-image: url(:/images/discord_hover.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGraphicsView" name="Logo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>138</width>
       <height>138</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/logo.png);
border:none;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Version">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>12</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Version&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Site">
     <property name="geometry">
      <rect>
       <x>1232</x>
       <y>60</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#Site {
  width: 32px;
  height: 32px;
  border-radius: 16px; /* Половина высоты кнопки для создания круглой формы */
  background-color: black;
  border: 2px solid #ffffff;
  padding: 0;
	background-image: url(:/images/globe.png);
  background-position: center;

}


#Site:hover:pressed {
    background-color: rgba(92, 184, 92, 80%); /* Прозрачный фон кнопки при нажатии */
}

#Site:hover:!pressed {
 background-color: black;
background-image: url(:/images/globe_hover.png);
  border: 2px solid #ffca00;
  padding: 0;
}


</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="Close">
     <property name="geometry">
      <rect>
       <x>1239</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Close {

  background-color: black;
 
}


#Close:hover:!pressed {
 background-color: red;

}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/close.png</normalon>
      </iconset>
     </property>
    </widget>
    <widget class="QProgressBar" name="file_progress_bar">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>600</y>
       <width>811</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#file_progress_bar {
    background-color: rgba(0, 0, 0, 0.5);
    border-radius: 5px;
}

#file_progress_bar::chunk {
    background-color: white;
    border-radius: 5px;
}

</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="overall_progress_bar">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>640</y>
       <width>811</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#overall_progress_bar {
    background-color: rgba(0, 0, 0, 0.5);
    border-radius: 5px;
}

#overall_progress_bar::chunk {
    background-color: white;
    border-radius: 5px;
}


</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
    </widget>
    <widget class="QTabWidget" name="info">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>551</width>
       <height>311</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane {
    border: 1px solid #C4C4C3;
    background: rgba(255, 255, 255, 0.0); /* Прозорий фон для вкладок */
}

QTabWidget::tab-bar {
    left: 5px; /* відступ від лівого краю */
}

QTabBar::tab {
    background: rgba(242, 242, 242, 0.0); /* Прозорий фон для вкладок */
    
    min-width: 8ex;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: rgba(250, 250, 250, 0.2);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: rgba(242, 242, 242, 0.2);
}

/* Додайте стиль для віджетів всередині вкладок */
QWidget {
    background: rgba(255, 255, 255, 0.0); /* Прозорий фон для віджетів */
}
</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Page</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QToolButton" name="Minimize">
     <property name="geometry">
      <rect>
       <x>1199</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Minimize {

  background-color: black;
 
}


#Minimize:hover:!pressed {
 background-color: #3a3939;

}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/minimize-sign.png</normalon>
      </iconset>
     </property>
    </widget>
    <widget class="QGraphicsView" name="Baner">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>240</y>
       <width>461</width>
       <height>213</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>461</width>
       <height>213</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
    </widget>
    <widget class="QWidget" name="BannerControls" native="true">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>425</y>
       <width>461</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
     </layout>
    </widget>
    <widget class="QPushButton" name="update_button">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>590</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="launch_button">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>590</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <zorder>Close</zorder>
    <zorder>Discord</zorder>
    <zorder>Facebook</zorder>
    <zorder>Instagram</zorder>
    <zorder>Logo</zorder>
    <zorder>Site</zorder>
    <zorder>Telegram</zorder>
    <zorder>VK</zorder>
    <zorder>Version</zorder>
    <zorder>info</zorder>
    <zorder>file_progress_bar</zorder>
    <zorder>overall_progress_bar</zorder>
    <zorder>Minimize</zorder>
    <zorder>Baner</zorder>
    <zorder>BannerControls</zorder>
    <zorder>update_button</zorder>
    <zorder>launch_button</zorder>
   </widget>
   <zorder>frame</zorder>
   <zorder>Name_server</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Close</sender>
   <signal>clicked(bool)</signal>
   <receiver>LauncherWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1252</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
